modules/apps/portal-security/portal-security-auth-test/src/testIntegration/java/com/liferay/portal/security/auth/test/AuthenticatedSessionManagerUtilTest.java

```diff
@@ -129,6 +134,16 @@ public void testRememberMeLogout() throws Exception {
 			CookiesConstants.CONSENT_TYPE_FUNCTIONAL, cookie,
 			mockHttpServletRequest, mockHttpServletResponse);
 
+		ThemeDisplay themeDisplay = ThemeDisplayFactory.create();
+
+		Group group = GroupTestUtil.addGroup();
+
+		themeDisplay.setCompany(CompanyTestUtil.addCompany());
```

Comment:
@manuelecastro is it necessary to create a whole new Company for this test? It bears a high cost.

Reply:
@stian-sigvartsen I added as [LogoutPreAction](https://github.com/liferay-appsec/liferay-portal/blob/8aecdeea188c7659808f4893f1adab1ebbbe48d9/modules/apps/click-to-chat/click-to-chat-web/src/main/java/com/liferay/click/to/chat/web/internal/events/LogoutPreAction.java#L40) requests `themeDisplay.getCompanyId()`. Is there a way we can escape this?

Reply:
I think so. I will push a commit